.coveragerc

```diff
@@ -1,12 +0,0 @@
-[run]
-# don't collect the plugins subdirectories
-omit =
-    plugins/*/env/*.py
-    plugins/conftest.py
-    plugins/*/*/tests/*
-    tests/plugins/*
-    sideboard/tests/*
-    tests/*
-
-include =
-    sideboard/*
```

Comment:
I believe that including this kind of thing in ``setup.cfg`` is considered more canonically correct these days, so this change seems good.